exs/mundo_1/ruby/009.rb

```diff
@@ -0,0 +1,16 @@
+=begin
+Desafio 009
+Problema: Faça um programa que leia um número Inteiro qualquer
+          e mostre na tela a sua tabuada.
+Resolução do problema:
+=end
```

Comment:
Mais uma de espaçamento

```suggestion
=begin
Desafio 009

Problema: Faça um programa que leia um número Inteiro qualquer
          e mostre na tela a sua tabuada.
          
Resolução do problema:
=end
```